examples/notebooks/keras_private_predict.ipynb

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
<p>using `clone_from` should allow us to avoid the call to `set_weights`</p>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>